packages/flame/lib/src/components/particle_component.dart

```diff
@@ -7,6 +7,7 @@ import 'component.dart';
 /// to a [Component] tree. Could be added either to FlameGame
 /// or an implementation of [Component].
 /// Proxies [Component] lifecycle hooks to nested [Particle].
+@Deprecated('Will be removed after v1.1, use ParticleSystemComponent instead')
 class ParticleComponent extends Component {
```

Comment:
This could have simply been 
```
class ParticleSystemComponent extends PositionComponent {
```

Reply:
Then we couldn't have deprecated it, it would have been a breaking change right? And as you said previously, the name was confusing.

Reply:
`typedef ParticleComponent = ParticleSystemComponent;` fixes it. I think we did this multiple times before, didn't we?

Reply:
It would work if it is only a name change, but the changes in the class will be breaking right?

Reply:
And eventually ParticleComponent will be removed só better to properly warn the user about that

Reply:
> And eventually ParticleComponent will be removed só better to properly warn the user about that

That wouldn't be a problem when having the deprecated typedef though.

Reply:
That is true. But either way the classes are not equivalent anymore. The new one is more in line with the rest of our components, so typedef don't work for this situation